### ISSUE TITLE
Error update.isFullyPopulated(setFileTimeToCheckin) on GET

### ISSUE THREAD

**Issue by johannperez** [Microsoft/team-explorer-everywhere#291]
Hi, I am trying to pull code from TFSVC repo using TEE CLC 14.134.0 on Mac and I am getting this error: An error occurred: java.lang.IllegalArgumentException: update.isFullyPopulated(setFileTimeToCheckin)

It does not matter if I try using eclipse TEE or from command line, same result.

Getting code from windows works just fine.

Any hints? 

Thanks!

**Comment by toddbrownmcis** [comment]
@johannperez I had the same issue with a solution in TFVC.  Getting source on Windows self-hosted TFS build agent worked fine but failed with this error message on macOS self-hosted TFS build agent.  I was able to **workaround** this issue using [Kevin is at Work](https://social.msdn.microsoft.com/Forums/en-US/1f38de7a-b2d8-42db-9f30-8ff457f58fd7/updateisfullypopulatedsetfiletimetocheckin?forum=tfsversioncontrol) answer.

Basically, I got all the latest source locally, then re-added to TFVC in a new location.  Now getting source from the new location in TFVC works on both agents.

**Comment by toddbrownmcis** [comment]
@johannperez some more information related to this issue...

It seems that the Visual Studio for Mac extension [Team Foundation Version Control for TFS and VSTS](https://docs.microsoft.com/en-us/visualstudio/mac/tf-version-control) adds new files to TFS without setting the file encoding.  I am currently using version  0.4.4.4 of the extension.

This causes TEE CLC 14.134.0 to throw the error: java.lang.IllegalArgumentException: update.isFullyPopulated(setFileTimeToCheckin)

You can find all files that are causing this error by running [tf info -recursive ...](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/properties-or-info-command?view=azure-devops) and searching for "File type:     0"

I'm setting the file encoding from Visual Studio on my Windows computer with [Configure Version Control File Encoding](https://docs.microsoft.com/en-us/azure/devops/server/admin/manage-file-types?view=azure-devops#config-file-encodings).
